client/vue.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   devServer: {
     proxy: {
       '/api/': {
-        target: 'https://emocean.dev',
+        target: 'http://localhost:5000',
```

Comment:
Are you certain about this (when this arrives at prod)?

Reply:
ok, i see. I`ll change

Reply:
done